java/src/jmri/server/json/package-info.java

```diff
@@ -147,6 +147,13 @@
  * Starting with 5.0.0, the JSON protocol version follows semantic version
  * rules, prior to that the version is just a major.minor version.
  * <dl>
+ * <dt>5.1.0 (JMRI 4.17.2)</dt>
```

Comment:
Since you are breaking contract, the JSON version must be 6.0.0 (the major number must be changed for a breaking change, not the minor number.

Also, this will be in 4.17.3; 4.17.2 is being built already.

Reply:
Done.